cmd/routers.go

```diff
@@ -83,34 +107,16 @@ func configureServerHandler(srvCmdConfig serverCmdConfig) (http.Handler, error)
 
 	// Initialize distributed NS lock.
 	if globalIsDistXL {
-		// Register storage rpc router only if its a distributed setup.
-		err := registerStorageRPCRouters(mux, srvCmdConfig)
-		if err != nil {
-			return nil, err
-		}
+		registerDistXLRouters(mux, srvCmdConfig)
+	}
 
-		// Register distributed namespace lock.
-		err = registerDistNSLockRouter(mux, srvCmdConfig)
-		if err != nil {
+	// Register web router only if its not disabled.
+	if !globalIsBrowserDisabled {
```

Comment:
to avoid the above double-negation, when read in English, we can call the variable globalIsBrowserEnabled and initialize it accordingly. These kind of option names confuse users. e.g, nfs.disable option in GlusterFS.